Library/Formula/txt2man.rb

```diff
@@ -1,20 +1,40 @@
-require 'formula'
-
 class Txt2man < Formula
-  homepage 'http://mvertes.free.fr/'
-  url 'http://mvertes.free.fr/download/txt2man-1.5.6.tar.gz'
-  sha1 'ef1392785333ea88f7e01f4f4c519ecfbdd498bd'
+  homepage "http://mvertes.free.fr/"
+  url "http://mvertes.free.fr/download/txt2man-1.5.6.tar.gz"
+  sha1 "ef1392785333ea88f7e01f4f4c519ecfbdd498bd"
 
-  depends_on 'gawk'
+  depends_on "gawk"
 
   def install
-    man1.install %W[bookman.1 src2man.1 txt2man.1]
-    bin.install %W[bookman src2man txt2man]
+    man1.install %w[bookman.1 src2man.1 txt2man.1]
+    bin.install %w[bookman src2man txt2man]
   end
 
   test do
-    system "#{bin}/txt2man", "-h"
-    system "#{bin}/src2man", "-h"
-    system "#{bin}/bookman", "-h"
+    # txt2man
+    (testpath/"test.txt").write <<-EOS.undent
+    A TITLE
+
+    blah blah blah
```

Comment:
Really pedantic but can you indent `EOS.undent`s by two characters more? Just makes them a little easier to follow. Thanks dude!